### ISSUE TITLE
Iterate over a sequence of numbers

### ISSUE THREAD

**Issue by ebussieres** [mbosecke/pebble#118]
Just like for tag in twig, it would be cool to iterate over a sequence of numbers

```javascript
{% for i in 0..10 %}
    * {{ i }}
{% endfor %}
```

**Comment by r0b0tbuilder** [comment]
Could a sequence of characters, namely letters, be iterated over as well?

```javascript
{% for i in a..z %}
    * {{ i }}
{% endfor %}
```

**Comment by ebussieres** [comment]
It seems twig supports it, bit I don't see many use cases where it could be useful

**Comment by ebussieres** [comment]
I implemented this (see #122 ).

**Comment by thomashunziker** [comment]
Actually I use it within unit tests. So its useful!

**Comment by ebussieres** [comment]
I've just added the iteration over character

```javascript
{% for i in 'a'..'e' %}
    {{ i }}
{% endfor %}
```

**Comment by r0b0tbuilder** [comment]
:+1: 

@thomashunziker Just wondering, how exactly do you use this in unit tests?

**Comment by thomashunziker** [comment]
I use it to generate some dummy data. I fill a table with multiple rows. The rows are always the same. I use it to test if the page break is working. The generated HTML is used to generate a PDF.

**Comment by mbosecke** [comment]
Implemented by #122. Thanks everyone.
